life/modernized/02-life.adoc

```diff
@@ -2244,6 +2249,9 @@ and then your eyes shall see the blessed work of the Lord fulfilled in its due t
 for he is God Almighty, and all-sufficient:
 therefore let every heart confide in his power.
 
+[.embedded-content-document.letter]
```

Comment:
This is tricky. This is actually a short document that she wrote at some point in her life that was inserted at the end of her journal. It begins a few pages up where it says "A Salutation of my endeared love, in God's Holy fear...."  I would probably just leave this "Salutation" unembedded... because its pretty long, and kinda like just a publication that was appended to her journal.